linkerd/app/outbound/src/lib.rs

```diff
@@ -76,9 +76,75 @@ impl Config {
             // Limits the time we wait for a connection to be established.
             .push_timeout(self.proxy.connect.timeout)
             .push(metrics.transport.layer_connect(TransportLabels))
+            .push(admit::AdmitLayer::new(prevent_loop.into()))
             .into_inner()
     }
 
+    /// Constructs a TCP load balancer.
+    pub fn build_tcp_balance<C, E, I>(
```

Comment:
is there any actual change here, or did this just move around?

Reply:
looks like some of the parameters were also removed?

Reply:
Oh, fallback was extracted from tcp_balance https://github.com/linkerd/linkerd2-proxy/pull/662/commits/c18b0aed9c92844a105ebe96e1059892b6bd1518